tests/api/test_account.py

```diff
@@ -1424,7 +1424,7 @@ def test_staff_create(
     )
 
 
-@patch("saleor.dashboard.emails._send_set_user_password_email_with_url.delay")
+@patch("saleor.core.emails._send_set_user_password_email_with_url.delay")
```

Comment:
All the above: maybe it would be better to move those tasks to `saleor.account.emails`?

Reply:
Done 👍